bin/commands/init/stc/index.ts

```diff
@@ -0,0 +1,125 @@
+import * as config from '../../../libs/config';
+import * as zoslib from '../../../libs/zos';
+import * as zosJes from '../../../libs/zos-jes';
+import * as zosdataset from '../../../libs/zos-dataset';
+
+
+export function execute(allowOverwrite: boolean = false) {
+
+  common.printLevel1Message(`Install Zowe main started task`);
+  
+  // constants
+  const COMMAND_LIST = std.getenv('ZWE_CLI_COMMANDS_LIST');
+  
+   let stcExistence: boolean;
+  
+  // validation
+  common.requireZoweYaml();
+  const ZOWE_CONFIG=config.getZoweConfig();
+  
+  // read prefix and validate
+  const prefix=ZOWE_CONFIG.zowe?.setup?.dataset?.prefix;
+  if (!prefix) {
+    common.printErrorAndExit(`Error ZWEL0157E: Zowe dataset prefix (zowe.setup.dataset.prefix) is not defined in Zowe YAML configuration file.`, undefined, 157);
+  }
+  // read PROCLIB and validate
+  const proclib=ZOWE_CONFIG.zowe.setup?.dataset?.proclib;
+  if (!proclib) {
+    common.printErrorAndExit(`Error ZWEL0157E: PROCLIB (zowe.setup.dataset.proclib) is not defined in Zowe YAML configuration file.`, undefined, 157);
+  }
+  // read JCL library and validate
+  const jcllib = zoslib.verifyGeneratedJcl(ZOWE_CONFIG);
+  if (!jcllib) {
+    return common.printErrorAndExit(`Error ZWEL0319E: zowe.setup.dataset.jcllib does not exist, cannot run. Run 'zwe init', 'zwe init generate', or submit JCL ${prefix}.SZWESAMP(ZWEGENER) before running this command.`, undefined, 319);
+  }
+
+  let security_stcs_zowe=ZOWE_CONFIG.zowe.setup?.security?.stcs?.zowe;
+  if (!security_stcs_zowe) {
+    common.printErrorAndExit(`Error ZWEL0157E: (zowe.setup.security.stcs.zowe) is not defined in Zowe YAML configuration file.`, undefined, 157);
+  }
+  let security_stcs_zis=ZOWE_CONFIG.zowe.setup?.security?.stcs?.zis;
+  if (!security_stcs_zis) {
+    common.printErrorAndExit(`Error ZWEL0157E: (zowe.setup.security.stcs.zis) is not defined in Zowe YAML configuration file.`, undefined, 157);
+  }
+  let security_stcsAux=ZOWE_CONFIG.zowe.setup?.security?.stcs?.aux;
+  if (!security_stcsAux) {
+    common.printErrorAndExit(`Error ZWEL0157E: (zowe.setup.security.stcs.aux) is not defined in Zowe YAML configuration file.`, undefined, 157);
+  }
+
+  [security_stcs_zowe, security_stcs_zis, security_stcsAux].forEach((mb: string) => {
+    // STCs in target proclib
+    stcExistence=zosdataset.isDatasetExists(`${proclib}(${mb})`);
+    if (stcExistence == true) {
+      if (allowOverwrite) {
+        // warning
+        common.printMessage(`Warning ZWEL0300W: ${proclib}(${mb}) already exists. This data set member will be overwritten during configuration.`);
+      } else {
+        // common.printErrorAndExit(`Error ZWEL0158E: ${proclib}(${mb}) already exists.`, undefined, 158);
+        // warning
+        common.printMessage(`Warning ZWEL0301W: ${proclib}(${mb}) already exists and will not be overwritten. For upgrades, you must use --allow-overwrite.`);
+      }
+    }
+  });
+
+  if (stcExistence == true && !allowOverwrite) {
```

Comment:
`stcExistence` is existence of `security_stcsAux` only. 

This might be confusing , if you delete `AUX` and rerun the command in faith only `AUX` will be copied, existing members will be overwritten (without `--allow-overwrite` flag):

```
Warning ZWEL0301W: MARTIN.ZOWE.PROCLIB($$ZOWE$$) already exists and will not be overwritten. For upgrades, you must use --allow-overwrite.
Warning ZWEL0301W: MARTIN.ZOWE.PROCLIB($$ZIS$$) already exists and will not be overwritten. For upgrades, you must use --allow-overwrite.
...
Submitting Job ZWEISTC
```

Reply:
https://github.com/zowe/zowe-install-packaging/pull/3718/commits/29d19b8c8d60c6d96dfcb7df10a597b29cc13e09